Gem/Code/Source/Automation/ScriptReporter.cpp

```diff
@@ -1103,7 +1103,7 @@ namespace AtomSampleViewer
             const AZStd::string screenshotWarningsLogLine = AZStd::string::format("Screenshot warnings: %u \n", scriptReport.m_screenshotWarningCount);
             const AZStd::string failedScreenshotsLogLine = "\nScreenshot test info below.\n";     
 
-            const AZStd::string projectPath = AZ::Utils::GetProjectPath();
```

Comment:
should this be a reference or is it intentionally copying the string?

Reply:
In this case it doesn't have to be reference. A copy is just fine.

Reply:
In other words, copy elision in action.